edb/server/args.py

```diff
@@ -1552,6 +1552,8 @@ def parse_args(**kwargs: Any):
         if kwargs['compiler_pool_mode'] is not CompilerPoolMode.MultiTenant:
             abort("must use --compiler-pool-mode=fixed_multi_tenant "
                   "in multi-tenant mode")
+        if kwargs['testmode']:
+            abort("cannot use --testmode in multi-tenant mode")
 
```

Comment:
One thing that is annoying here is that `--testmode` is only passed at bootstrap time, but multitenant won't work on any database that was created with `--testmode`.

Sigh.

Reply:
Yeah, we can potentially load the stdschema from instdata for each tenant and verify its compatibility (by version?) with the stdlib loaded in multitenant server/compiler, and raise an appropriate error when they don't match. This might be useful regardless of how we deal with the testmode, though it's not a blocker as the main use case of multitenant server is the Cloud ourselves - the `--multitenant-config-file` option is still hidden from the general public.